src/index.js

```diff
@@ -47,6 +71,15 @@ const exec = (cwd, opts) => new Promise((resolve, reject) => {
 	})
 });
 
+const checkNpmCiExist = () => exec('npm ci --help')
+.then(output => output.toString() === 'npm ci');
+
+const doesPackageJsonExist = () => new Promise((resolve) => {
```

Comment:
Would `doesPackageLockJsonExist` better describe what this is doing?

Reply:
definitely